process/transaction/process.go

```diff
@@ -72,6 +68,9 @@ func NewTxProcessor(
 	if txTypeHandler == nil || txTypeHandler.IsInterfaceNil() {
 		return nil, process.ErrNilTxTypeHandler
 	}
+	if economicsFee == nil {
```

Comment:
Maybe you can also create IsInterfaceNil method and check in the same way as the other interfaces are checked here for nil

Reply:
Done.